### PR TITLE
Modified injected HTML to be XML-serializable (#559)

### LINKED ISSUE #559: Generated HTML is not XML-serializable
Currently the HTML generated by sweetalert2 is not valid XML, which causes issues when used in XML-serialized HTML (such as XHTML). I will try to create a PR to fix this.

### PR DESCRIPTION
**Pull request by Spetnik**
Fixes #559 
Added self-closing tags to void elements and replaced &times; with &#215;

### CODE REVIEW

File: src/utils/dom.js [hunk 1]
```diff
@@ -101,27 +101,27 @@ const sweetHTML = `
...
+     <input type="checkbox" />
    </label>
    <textarea class="${swalClasses.textarea}"></textarea>
    <div class="${swalClasses.validationerror}"></div>
    <div class="${swalClasses.buttonswrapper}">
      <button type="button" class="${swalClasses.confirm}">OK</button>
      <button type="button" class="${swalClasses.cancel}">Cancel</button>
    </div>
-   <button type="button" class="${swalClasses.close}" aria-label="Close this dialog">&times;</button>
+   <button type="button" class="${swalClasses.close}" aria-label="Close this dialog">&#215;</button>
```
Comment (limonte) [inline src/utils/dom.js]: We can replace `&#215;` with just `×` to improve the code readability.

**Comment by limonte** [comment]
Thanks a lot @Spetnik, just a small change to do before merge

**Comment by limonte** [comment]
Any objections from collaborators?

**Comment by toverux** [comment]
No objections, it's probably a rare use case, but why not! It still makes Swal2 better.

**Pull request merged**